corpus/accounts/models.py

```diff
@@ -151,3 +155,22 @@ def save(self, *args, **kwargs):
 
     def __str__(self):
         return f"{self.user.first_name} {self.user.last_name} [{self.sig.name}]"
+
+
+class Core(models.Model):
+    user = models.OneToOneField(User, null=False, on_delete=models.CASCADE)
+    post = models.CharField(max_length=100, null=False)
```

Comment:
Do you think we can define a finite set of roles and make this `CharField` have a `choices` attribute? I worry that people would be able to misspell core roles and it'll reflect directly on the Team page.